fearless/Modules/Staking/ValidatorInfo/ViewModel/ValidatorInfoViewModel.swift

```diff
@@ -0,0 +1,62 @@
+import UIKit
+import FearlessUtils
+import SoraFoundation
+
+protocol ValidatorInfoAccountViewModelProtocol {
+    var name: String? { get }
+    var address: String { get }
+    var icon: UIImage? { get }
+}
+
+protocol ImageWithTitleViewModelProtocol {
+    var image: UIImage { get }
+    var title: String { get }
+}
+
+struct ValidatorInfoAccountViewModel: ValidatorInfoAccountViewModelProtocol {
+    let name: String?
+    let address: String
+    let icon: UIImage?
+}
+
+struct EmptyStakeViewModel: ImageWithTitleViewModelProtocol {
+    let image: UIImage
+    let title: String
+}
+
+struct StakingAmountViewModel {
+    let title: String
+    let balance: BalanceViewModelProtocol
+}
+
+enum ValidatorInfoViewModel {
+    static let headerHeight: CGFloat = 52.0
+    static let rowHeight: CGFloat = 48.0
+    static let accountRowHeight: CGFloat = 56.0
+    static let emptyStakeRowHeight: CGFloat = 140.0
+
+    enum StakingRow {
+        case totalStake(LocalizableResource<StakingAmountViewModel>)
```

Comment:
LocalizableResource is everywhere. Maybe just pass [LocalizableResource<ValidatorInfoViewModel>] ?

Reply:
Can't quite understand what you mean. Do you suggest to change view model?

Reply:
`enum ValidatorInfoViewModel` rename to `enum ValidatorInfoSection`
and somewhere in VC:
`var sections: [LocalizableResource<ValidatorInfoSection>] = []`